src/main/java/com/cisco/trex/stateful/api/lowlevel/ASTFProgram.java

```diff
@@ -447,9 +483,19 @@ public void jmpNz(String varId, String label) {
   /**
    * get the total send bytes of the program
    *
-   * @return sent bytes
+   * @return sent bytes @Deprecated use getTotalSendBytesLong instead
    */
+  @Deprecated
   public int getTotalSendBytes() {
+    return (int) totalSendBytes;
+  }
```

Comment:
can just delete this method instead of deprecation?
the cast from long to int may lead a wrong result.

Reply:
i wanted to keep it to be backwards compatible, risk of giving a compilation error if i remove it

Reply:
okay.